### ISSUE TITLE
WDS trainer presentation (PPT)

### ISSUE THREAD

**Issue by DawnmarieDesJardins** [Microsoft/MCW-Big-data-and-visualization#26]
Slide 21 is missing alt text.

**Comment by mrfalafel** [comment]
Alt Text was added and checked in for Slide 21

**Comment by DawnmarieDesJardins** [comment]
Closing issue, addressed in August 2018 test/fix, now merged to master.
